packages/module/patternfly-docs/content/extensions/component-groups/examples/CullingInfo/CullingInfo.md

```diff
@@ -0,0 +1,37 @@
+---
+# Sidenav top-level section
+# should be the same for all markdown files
+section: Component groups
+subsection: Status and state indicators
+# Sidenav secondary level section
+# should be the same for all markdown files
+id: Culling information
```

Comment:
```suggestion
id: Status timestamp
```

Previously, ([via this Miro](https://miro.com/app/board/uXjVKr2og3k=/)), we considered renaming "culling info" to be "status timestamp" for easier understanding. I don't see a timestamp in the examples, though, so maybe the design has changed. Do we still want to do that?

Reply:
Another option could be "Stale warning", "Expiration warning", "Stale data warning"

Reply:
I'm not able to access the Miro link, but there isn't a timestamp. It is either a warning/danger tooltip or a warning/danger icon with a custom message. Is there a naming option you would prefer?

Reply:
Okay, let's go with "stale data warning"  if that sounds okay with you? I also don't mind sticking with "culling" if that's more accurate, but if so I would suggest "culling warning" instead of "culling information"